spring-auth/src/main/kotlin/no/nav/poao/dab/spring_auth/AuthService.kt

```diff
@@ -118,3 +145,40 @@ class AuthService(
         return authContextHolder.requireIdTokenString();
     }
 }
+
+
+interface IResoult {
+    val harTilgang: Boolean
+    fun thorwIfIkkeTilgang()
+    fun getLogbuilder(): CefMessageBuilder?
+
+}
+
+data class Resoult(override val harTilgang: Boolean, val accesedIdnet: Id, val byIdent: Id, val melding: String? = null) :
+    IResoult {
+    override fun thorwIfIkkeTilgang() {
```

Comment:
```suggestion
    override fun throwIfIkkeTilgang() {
```
Må refaktoreres for å endre alle plasser den blir brukt